.circleci/config.yml

```diff
@@ -813,25 +804,19 @@ jobs:
     docker:
       - image: << pipeline.parameters.ubuntu-2004-clang-docker-image >>
     steps:
-      - checkout
-      - attach_workspace:
-          at: build
-      - run: *run_soltest
+      - when:
+          condition: true
+          <<: *steps_soltest
```

Comment:
This I don't quite understand... is this just because ``<<: *steps_soltest`` on the outermost level would be invalid for some reason?

Reply:
The reason is that putting it at the same level as `steps` would make it completely ovewrite `steps` rather than add keys to it.

I tried to explain it in one of the recent PRs: https://github.com/ethereum/solidity/pull/11823#discussion_r696042121

Reply:
That makes sense, but there is no other ``steps`` key here, is there? So what's the problem with overwriting what does not exist :-)? But I may be missing something and I don't mind in any case :-).

Reply:
Ah sorry, I just missed the ``- run: *gitter_notify_failure`` thing below, so indeed makes perfect sense.